src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/services/CancellationTaskHandlerServiceImpl.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.wacaseeventhandler.services;
+
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Service;
+
+@Service
+@Order(1)
+public class CancellationTaskHandlerServiceImpl implements CaseEventHandlerService {
```

Comment:
Personally I'd prefer calling them `CaseEventHandler` and `CancellationTaskHandler` etc, what do you think?

Not a fan of `Impl` as it usually means we could have named it better.  E.g. if you consider the interface `List` and it's implementations.

Reply:
Yes, agree, I let the IDE make that decision for me :)  I will change it.

Reply:
Done. It looks much nicer now. Thanks!